### PR TITLE
Update cairo.rb

### PR DESCRIPTION
**Pull request by lovedheart**
cairo-xlib is requied to build gtk

#889

### CODE REVIEW

File: packages/cairo.rb [hunk 1]
```diff
@@ -7,32 +8,17 @@ class Cairo < Package
...
   depends_on 'fontconfig' # pango requires cairo with fontconfig
+  depends_on 'gobject_introspection'
+  depends_on 'mesa'
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--disable-xlib'
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --enable-xlib --enable-xlib-xcb --enable-glesv2"
```
Comment (uberhacker) [inline packages/cairo.rb]: Nice!  Can you please keep the separation of one configure option per line?  It just makes it easier to examine and add/remove options in the future.

File: packages/cairo.rb [hunk 2]
```diff
@@ -7,32 +8,17 @@ class Cairo < Package
...
     system "make"
   end
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
   end
-end
```
Comment (uberhacker) [inline packages/cairo.rb]: Don't we need the ending `end` here?

File: packages/cairo.rb (hunk 1 above)
Comment (lovedheart) [reply packages/cairo.rb]: I got error if I do that. I don't know why

**Comment by uberhacker** [comment]
@lovedheart: I really appreciate your efforts to add all these new packages and improvements to existing packages but please test before submitting PRs.  Most the packages you submitted earlier, I wasn't able to compile either due to missing dependencies or missing packages altogether.  If you test before you commit, it makes it much easier on me since I have to generate binaries.
Thanks

File: packages/cairo.rb (hunk 1 above)
Comment (uberhacker) [reply packages/cairo.rb]: You need a trailing comma on each line or you can concatenate with a trailing `\`.  Also, you need all the crew constants to be enclosed in double quotes or the variables won't expand.  Hope that helps.

File: packages/cairo.rb [hunk 3]
```diff
@@ -7,28 +8,14 @@ class Cairo < Package
   source_url 'https://www.cairographics.org/releases/cairo-1.14.12.tar.xz'
```
Comment (uberhacker) [inline packages/cairo.rb]: Please bump the version to detect the changes, otherwise `crew update` will not update previous installs of cairo.

Comment (lovedheart) [reply packages/cairo.rb]: How to do that?

Comment (uberhacker) [reply packages/cairo.rb]: `version '1.14.12-1'` should do it.

File: packages/cairo.rb (hunk 1 above)
Comment (uberhacker) [reply packages/cairo.rb]: Please change to:
```
system "./configure \
  --prefix=#{CREW_PREFIX} \
  --libdir=#{CREW_LIB_PREFIX} \
  --enable-xlib \
  --enable-xlib-xcb \
  --enable-glesv2"
```

**Comment by uberhacker** [comment]
Please email a subscribe message to `crewonslack@gmail.com` and I will send you an invite to join our Slack group.

**Pull request merged**